packs/classfeatures/heavy-construction.json

```diff
@@ -17,7 +17,11 @@
             "value": 7
         },
         "prerequisites": {
-            "value": []
+            "value": [
+                {
+                    "value": "Power Suit Only"
```

Comment:
We probably want to be consistent on whether these parenthesis are added as prerrequisites or as part of the description like it's done elsewhere in this PR. I'm leaning toward making it part of the description as these aren't prerrequisites in the same way feats have prerrequisites. 
![image](https://github.com/foundryvtt/pf2e/assets/103401186/0abcf519-187d-4318-9886-08ad9b6fdb86)

Reply:
Fair enough - I'll change these to be part of the description then.

Reply:
If we want to be consistent, omit them and use the predicates in the (future) Choice Set for Revolutionary Innovation to filter them out

Reply:
OK, those changes should be in now. Please check when you get a moment. 

Thanks :)

Reply:
As I said, I think you're better off omitting these requirements for the sake of consistency with existing modifications. The ChoiceSet can do the filtering for prerequisites for you.

Reply:
Weird - I see them omitted on my end.

Reply:
After checking, it looks like we don't do this consistently in the system. Sometimes the prerequisites are in the body of the text, sometimes they are omitted. This looks fine to me, but ultimately up to one of the leads which way we actually go with.